test/data_map_test.exs

```diff
@@ -0,0 +1,162 @@
+defmodule DataMapTest do
+  use PowerAssert
+
+  test "Insert empty row must fail" do
+    c1 = %Masdb.Schema.Column{is_pk: true,  name: "c1", type: :int}
+    c2 = %Masdb.Schema.Column{is_pk: false, name: "c2", type: :int}
+    c3 = %Masdb.Schema.Column{is_pk: false, name: "c3", type: :int}
+
+    timestamp = Masdb.Timestamp.get_timestamp()
+    inserted_value = %{}
+    inserted_schema = %Masdb.Schema{name: "foo", replication_factor: 0, columns: [c1, c2, c3]}
+    inserted_nodeid = "foo@127.0.0.1"
```

Comment:
I'm not quite sure, because we were taking the string given by inspect(Node.self).
To be correct, it would be ":foo@127.0.0.1" but I think it has no importance